mousestyles/data/__init__.py

```diff
@@ -62,3 +62,105 @@ def load_all_features():
     other_features = [x.iloc[:, -1] for x in data_frames]
     other_features.insert(0, all_features_df)
     return pd.concat(other_features, axis=1)
+
+
+def load_intervals(feature):
+    """
```

Comment:
Please add a short 1 sentence summary of each function at the top of the docstring.  While the function name should be as self-explanatory as possible, it is often a useful exercise to expand the encoded function name into a short sentence or phrase.  This can help you think about whether the name of the function is appropriate.  It also provides a bit more clarity to the authors intention, which is helpful for users and collaborators.

For example, 
"Return the project interval data for the specified feature."